chartfx-chart/src/main/resources/de/gsi/chart/chart.css

```diff
@@ -4,165 +4,192 @@
     -fx-border-radius: 3.0;
     -fx-font-size: 12.0;
     -fx-font-weight: bold;
-	-fx-text-alignment: center;
+    -fx-text-alignment: center;
 }
 
 .axis-label {
     -fx-font-size: 13.0;
-    -fx-font-weight: normal;    
+    -fx-font-weight: normal;
     /*-fx-stroke: derive(-fx-background,-30%);*/
     -fx-stroke: rgb(255, 0, 0);
-	/*-fx-text-alignment: center;*/
-	-fx-axis-label-alignment: center;
+    /*-fx-text-alignment: center;*/
+    -fx-axis-label-alignment: center;
 }
 
 .chart-crosshair-path {
-	-fx-stroke-width: 1.0;
+    -fx-stroke-width: 1.0;
 }
 
 .chart-zoom-rect {
-	-fx-fill: dodgerblue;
-	-fx-stroke: #002966 ;
-	-fx-stroke-type: inside;
-	-fx-stroke-width: 1.0;
-	-fx-opacity: 0.2;
+    -fx-fill: dodgerblue;
+    -fx-stroke: #002966;
+    -fx-stroke-type: inside;
+    -fx-stroke-width: 1.0;
+    -fx-opacity: 0.2;
 }
 
 .chart-select-rect {
-	-fx-fill: transparent;
-	-fx-stroke: gray ;
-	-fx-stroke-width: 1.5;
-	-fx-stroke-dash-offset: 6;
-	-fx-stroke-dash-array: 12 2 4 2;
-	-fx-stroke-line-cap: butt;
-	 -fx-effect: dropshadow(three-pass-box, derive(gray, -20%), 10, 0, 4, 4);	 
+    -fx-fill: transparent;
+    -fx-stroke: gray;
+    -fx-stroke-width: 1.5;
+    -fx-stroke-dash-offset: 6;
+    -fx-stroke-dash-array: 12 2 4 2;
+    -fx-stroke-line-cap: butt;
+    -fx-effect: dropshadow(three-pass-box, derive(gray, -20%), 10, 0, 4, 4);
 }
 
 .chart-select-marker {
-	-fx-fill: transparent;
-	-fx-stroke: lightGreen ;
-	-fx-stroke-width: 1.5;
-	-fx-stroke-dash-offset: 6;
-	-fx-stroke-dash-array: 12 2 4 2;
-	-fx-stroke-line-cap: butt;
-	-fx-opacity: 1.0;
-	-fx-effect: dropshadow(three-pass-box, green, 5, 0.5, 0, 0);
+    -fx-fill: transparent;
+    -fx-stroke: lightGreen;
+    -fx-stroke-width: 1.5;
+    -fx-stroke-dash-offset: 6;
+    -fx-stroke-dash-array: 12 2 4 2;
+    -fx-stroke-line-cap: butt;
+    -fx-opacity: 1.0;
+    -fx-effect: dropshadow(three-pass-box, green, 5, 0.5, 0, 0);
 }
 
 .chart-select-marker:noEdit {
-	-fx-fill: transparent;
-	-fx-stroke: red ;
-	-fx-stroke-width: 1.5;
-	-fx-stroke-dash-offset: 6;
-	-fx-stroke-dash-array: 12 2 4 2;
-	-fx-stroke-line-cap: butt;
-	-fx-opacity: 1.0;
-	-fx-effect: dropshadow(three-pass-box, red, 5, 0.5, 0, 0);
+    -fx-fill: transparent;
+    -fx-stroke: red;
+    -fx-stroke-width: 1.5;
+    -fx-stroke-dash-offset: 6;
+    -fx-stroke-dash-array: 12 2 4 2;
+    -fx-stroke-line-cap: butt;
+    -fx-opacity: 1.0;
+    -fx-effect: dropshadow(three-pass-box, red, 5, 0.5, 0, 0);
 }
 
-.chart-series-line {    
+.chart-series-line {
     -fx-stroke-width: 1.0px;
     -fx-effect: null;
 }
 
-.value-indicator-label{
+.value-indicator-label {
     -fx-background-color: #f4e242;
     -fx-border-color: black;
     -fx-border-radius: 2.0;
     -fx-font-size: 12.0;
     -fx-font-weight: bold;
-	-fx-text-alignment: center;
-	-fx-padding: 1.0 2.0 1.0 2.0;
+    -fx-text-alignment: center;
+    -fx-padding: 1.0 2.0 1.0 2.0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780860)